lib/GithubHandler.js

```diff
@@ -177,15 +177,15 @@ GithubHandler.prototype.pullRequestHandler = function(event, cb) {
     type: 'pull_request', // also in event.event
     service: 'github',
     host: undefined,
+    branch: event.payload.pull_request.head.ref,
```

Comment:
Should this specify source ref and should we be passing in target ref in order to do a download of the merged code rather than this specific commit?